### ISSUE TITLE
OSX: Sourcetrail.2018_2.19: signature error

### ISSUE THREAD

**Issue by romadinov** [CoatiSoftware/SourcetrailBugTracker#568]
Hi,

My OSX complains about Sourcetrail 2018.2.19 with a message ""Sourcetrail" can't be opened because the identity of the developer cannot be confirmed." It looks like there is a signature error

**Comment by egraether** [comment]
There is currently an issue with our macOS Developer Application ID. The codesigning succeeds, but macOS is complaining that the developer is not identified. I contacted Apple Developer Support regarding this and I'm still waiting for a reply.

You can run Sourcetrail via "right click + Open"

**Comment by egraether** [comment]
Fixed with 2018.2.31

For future reference to anyone with the same problem (and myself):

Response from Apple Developer Support:
```
Thank you for contacting Apple Developer Technical Support (DTS).

Based on your request Apple Developer Technical Support believes that your question is answered by the "Gatekeeper Changes in macOS 10.10.4 and Later" section of "Technical Note TN2206: macOS Code Signing In Depth" linked here:

https://developer.apple.com/library/content/technotes/tn2206/_index.html#//apple_ref/doc/uid/DTS40007919-CH1-TNTAG207
```

So the `codesign` and `spctl` tools didn't show me the error, but I found it in the system log. It turned out there was an absolute Qt loader path left in the `sourcetrail_indexer` binary, which caused macOS Gatekeeper to refuse the bundle.

This path was not visible in `otool -L <path/to/binary>`:
```
sourcetrail_indexer:
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1452.23.0)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/usr/lib/libncurses.5.4.dylib (compatibility version 5.4.0, current version 5.4.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.50.4)
	@rpath/QtWidgets.framework/Versions/5/QtWidgets (compatibility version 5.10.0, current version 5.10.1)
	@rpath/QtGui.framework/Versions/5/QtGui (compatibility version 5.10.0, current version 5.10.1)
	@rpath/QtNetwork.framework/Versions/5/QtNetwork (compatibility version 5.10.0, current version 5.10.1)
	@rpath/QtCore.framework/Versions/5/QtCore (compatibility version 5.10.0, current version 5.10.1)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 400.9.0)
```

It was only visible with `otool -l <path/to/binary>`:
```
...
Load command 21
          cmd LC_RPATH
      cmdsize 64
         path /Users/ebsi/Documents/Qt/5.10.1/clang_64/lib (offset 12)
...
```

Deleting the path fixed the issue:
```
install_name_tool -delete_rpath "/Users/ebsi/Documents/Qt/5.10.1/clang_64/lib" <path/to/binary>
```
